src/util.cr

```diff
@@ -33,5 +33,10 @@ module Wsman
         log.info("Not removing #{path}, does not exist.")
       end
     end
+
+    def self.get_gid_for(name)
```

Comment:
We can create a `User` class later, when we want more control over users in Crystal.